lib/executor.js

```diff
@@ -116,7 +116,9 @@ function setPort(app, instructions) {
     process.env.npm_package_config_port,
     app.get('port'),
     3000
-  ], _.isFinite);
+  ], function(v) {
+    return _.isFinite(parseInt(v, 10));
+  });
```

Comment:
> Note that parseInt('10asd', 10) return 10, which is probably not what we want.

I take that back, the purpose of the port-scanning code is to find the first value that is defined.

In order to keep the size of the the browser bundle small, we are trying to get rid of lodash/underscore in as many places as possible. If `_.isFinite` in the new version of lodash does not work in the same way as the old one, then I would much rather rewrite this condition so that it does not depend on lodash at all.

I think the following may serve pretty well:

```js
function isNumberLike(v) { return Number.isFinite(parseInt(v, 10)); }
```